library/std/src/collections/hash/map.rs

```diff
@@ -3109,7 +3107,7 @@ impl<'a, K: 'a, V: 'a> VacantEntry<'a, K, V> {
     /// assert_eq!(map["poneyland"], 37);
     /// ```
     #[inline]
-    #[unstable(feature = "entry_insert", issue = "65225")]
+    #[stable(feature = "entry_insert", since = "1.82.0")]
```

Comment:
This should probably be `since = "CURRENT_RUSTC_VERSION"`

Reply:
And the other place too

Sadly, I didn't manage to figure out how to do GitHub's fancy patch-like suggestions.

Reply:
done, thanks

Reply:
> I didn't manage to figure out how to do GitHub's fancy patch-like suggestions.

You can insert a suggestion using either Ctrl+G or this button:
![Add a suggestion button](https://github.com/user-attachments/assets/25d8def7-e98b-4b31-bf91-6a8503b6441a)

You can then edit the code block to represent your suggested change:
````markdown
```suggestion
    #[stable(feature = "entry_insert", since = "CURRENT_RUSTC_VERSION")]
```
````